client/src/Form.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '' };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ value: event.target.value });
+  }
+
+  handleSubmit(event) {
+    alert('A name was submitted: ' + this.state.value);
+    event.preventDefault();
+  }
+  render() {
+    return (
+      <div>
+        <form onSubmit={this.handleSubmit}>
+          <label>
+            Name:
+          <input type="text" value={this.state.value} onChange={this.handleChange} />
+          </label>
+          <label>
+            number:
+          <input type="text" value={this.state.value} onChange={this.handleChange} />
+          </label>
+          <label>
+            Email:
+          <input type="text" value={this.state.value} onChange={this.handleChange} />
+          </label>
+          <label>
+            Message:
+          <input type="text" value={this.state.value} onChange={this.handleChange} />
+          </label>
+          <input type="submit" value="Contact Agent" />
+        </form>
+      </div>
+    )
+  }
+}
+
+export default Form;
```

Comment:
A few changes here. Your state needs a way to store individual values from each input box. I would recommend something like this. 

        this.state = { 
            information: {}
        }
information is going to store the values of each of your inputs as a property. This is accomplished by using the name attribute in your html element. For example 

                <input type="text" className="form-control" id="inputZip" **name="zip"** onChange={changeInfo}/>
       
and then refactor your handle change method to something like this. 

handleChange(e) {
        const target = event.target; 
        var value = target.value; // grab the value from target input
        const name = target.name; //grab the name from the target input
        const information = this.state.information; // grab the current information obj from state
        information[`${name}`] = value // create a new key value pair
        this.setState({ 
            information: information 
        }); // set state
    };

Reply:
updated